src/garage/tf/algos/_utils.py

```diff
@@ -0,0 +1,33 @@
+"""Utility functions used by tensorflow algorithms."""
```

Comment:
please put this in a module `_functions.py` instead (we are trying to move to this pattern across the codebase

Reply:
Do you mean the module in `src/garage/_functions.py`? or simply create a `_functions.py` under `src/garage/tf`?

The interface is very similar to the one in torch except it accepts optimizer type only, not tuple. And if we decide to use `src/garage/_functions.py`, I can move both tf, torch `make_optimizer` functions under the interface

Reply:
If your function needs tf, put it in `src/garage/tf/_functions.py` and export to `garage.tf`

If your function is not specific to tf (and can be share by torch), please move the shared implementation to `src/garage/_function.py` and export to `garage` package.

Reply:
I refactored the tf and pytorch shared implementation and moved them to `src/garage/_function.py` for export.

Reply:
Did you upload your new version yet? I don't see it here